action.yml

```diff
@@ -42,6 +42,10 @@ inputs:
     default: true
     required: false
   
+  TIME_RANGE:
+    description: "Time range of the queried statistics"
+    default: "last_7_days"
```

Comment:
I think `required: false` is necessary..

Reply:
I think it's not necessary, since the default value is set if the value is not provided.

Reply:
ok..